holidayextras/ui-toolkit
# Update browserify from 10.2.6 to 11.0.1

#### What does this PR do? (please provide any background)
Updates browserify to the latest version to resolve the dependencies out of date badge

#### What tests does this PR have?
N/A

#### How can be this tested?
```
npm install
grunt docs
```
Play with the docs & check it behaves as you'd expect

#### Screenshots / Screencast
![dependencies](https://cldup.com/Th1O0_q2F2.png)

#### What gif best describes how you feel about this work?
![chicken](http://media.giphy.com/media/syCa5ird7wp0c/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
